addons/tmf_loadouts/loadouts/US_Army_2018.hpp

```diff
@@ -0,0 +1,576 @@
+		"ACE_Flashlight_XL50",
+		"ACE_IR_Strobe_Item",
+		"ACE_microDAGR"
+	};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch"
+	};
+
+	backpackItems[] = {};
+
+	traits[] = {};
+
+	code = "";
+};
+
+class r : baseMan {
+	displayName = "Rifleman";
+	vest[] = {
+		"CUP_V_B_IOTV_OCP_Rifleman_USArmy",
+		"CUP_V_B_IOTV_OCP_Rifleman_Deltoid_USArmy"
+	};
+	headgear[] = {
+		"CUP_H_USArmy_HelmetACH_OCP",
+		"CUP_H_USArmy_HelmetACH_GCOVERED_OCP",
+		"CUP_H_USArmy_HelmetACH_GCOVERED_Headset_OCP",
+		"CUP_H_USArmy_HelmetACH_ESS_OCP",
+		"CUP_H_USArmy_HelmetACH_ESS_Headset_OCP",
+		"CUP_H_USArmy_HelmetACH_Headset_OCP"
+	};
+	primaryWeapon[] = {
+		"CUP_arifle_M4A1_black",
+		"CUP_arifle_M4A3_black"
+	};
+	scope[] = {
+		"cup_optic_eotech553_black",
+		LIST_2("cup_optic_compm4")
+	};
+	attachment[] = {
+		"cup_acc_anpeq_15",
+		"cup_acc_anpeq_15_tan_top"
+	};
+	magazines[] = {
+		LIST_8("CUP_30Rnd_556x45_Stanag"),
+		LIST_2("CUP_30Rnd_556x45_Stanag_Tracer_Red"),
+		LIST_2("HandGrenade"),
+		LIST_2("SmokeShell")
+	};
+};
+
+class g : r {
+	displayName = "Grenadier";
+	vest[] = {"CUP_V_B_IOTV_OCP_Grenadier_USArmy"};
+	backpack[] = {"CUP_B_US_Assault_OCP"};
+	primaryWeapon[] = {"CUP_arifle_M4A1_BUIS_GL"};
+	attachment[] = {"cup_acc_anpeq_15"};
+	magazines[] += {
+		LIST_6("CUP_30Rnd_556x45_Stanag"),
+		LIST_2("CUP_30Rnd_556x45_Stanag_Tracer_Red"),
+		"HandGrenade",
+		"SmokeShell",
+		LIST_8("CUP_1Rnd_HEDP_M203"),
+		LIST_2("CUP_1Rnd_Smoke_M203")
+	};
+};
+
+class car : r {
+	displayName = "Carabinier";
+};
+
+class cls : car {
+	displayName = "Combat Life Saver";
+	traits[] = {"medic"};
+	vest[] = {"CUP_V_B_IOTV_OCP_Medic_USArmy"};
+	backpack[] = {"CUP_B_US_Assault_OCP"};
+	backpackItems[] = { MEDICAL_CLS };
+};
+
+class m : cls {
+	displayName = "Medic";
+	backpack[] = {"CUP_B_US_IIID_OCP"};
+	backpackItems[] = { MEDICAL_M };
+};
+
+class smg : r {
+	displayName = "Submachinegunner";
+	vest[] = {"CUP_V_B_IOTV_OCP_Empty_USArmy"};
+	primaryWeapon[] = {"CUP_arifle_M4A1_standard_short_black"};
+	magazines[] = {
+		LIST_6("CUP_30Rnd_556x45_Stanag"),
+		"HandGrenade",
+		"SmokeShell"
+	};
+};
+
+class ftl : g {
+	displayName = "Fireteam Leader";
+	vest[] = {"CUP_V_B_IOTV_OCP_TL_USArmy"};
+	backPack[] = {"CUP_B_US_Assault_OCP"};
+	items[] += {"ACE_MapTools"};
+	magazines[] += {
+		LIST_2("CUP_1Rnd_SmokeRed_M203"),
+		LIST_2("CUP_1Rnd_SmokeGreen_M203"),
+		LIST_2("CUP_1Rnd_StarFlare_Red_M203")
+	};
+	linkedItems[] += {"ACE_Yardage450"};
+};
+
+class sl : r {
+	displayName = "Squad Leader";
+	vest[] = {"CUP_V_B_IOTV_OCP_SL_USArmy"};
+	backPack[] = {"CUP_B_US_Assault_OCP"};
+	sidearmWeapon[] = {"CUP_hgun_M17_Coyote"};
+	items[] += {"ACE_MapTools"};
+	magazines[] += {
+		LIST_3("CUP_21Rnd_9x19_M17_Coyote"),
+		LIST_2("SmokeShellGreen"),
+		LIST_2("SmokeShellRed")
+	};
+	linkedItems[] = {"ACE_Yardage450"};
+};
+
+class co : sl {
+	displayName = "Platoon Leader";
+	headgear[] = {"CUP_H_US_patrol_cap_OCP"};
+};
+
+class fac : ftl {
```

Comment:
As you're inheriting from FTL, might as well throw on some more smoke rounds for the GL.